.github/workflows/presubmit.yaml

```diff
@@ -0,0 +1,111 @@
+name: Presubmit
+
+on:
+  pull_request:
+  push:
+    branches-ignore:
+      # push events to main branch occur after PRs are merged, when the same checks were run
+      - main
+  workflow_dispatch:
+
+jobs:
+  build-linux-x86_64-with-gfxstream:
+    name: build rutabaga with gfxstream (Linux x86_64)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: stable
+          targets: x86_64-unknown-linux-gnu
+      - uses: ./.github/actions/install-deps
+      - name: Checkout gfxstream
+        run: git clone --depth 1 https://github.com/google/gfxstream
```

Comment:
Should we checkout at a specific commit? Otherwise, I suppose errors in upstream might show up as errors here and block this project.